src/test/java/reposense/template/GitTestTemplate.java

```diff
@@ -18,24 +20,34 @@
 import reposense.authorship.model.LineInfo;
 import reposense.git.GitCheckout;
 import reposense.git.GitClone;
+import reposense.git.GitShow;
+import reposense.git.exception.CommitNotFoundException;
 import reposense.model.Author;
 import reposense.model.CommitHash;
 import reposense.model.FileTypeTest;
 import reposense.model.RepoConfiguration;
 import reposense.model.RepoLocation;
+import reposense.util.FileUtil;
 
 /**
  * Contains templates for git testing.
  */
+
 public class GitTestTemplate {
-    protected static final String TEST_REPO_GIT_LOCATION = "https://github.com/reposense/testrepo-Alpha.git";
+    protected static final String TEST_REPO_GIT_LOCATION = "https://github.com/FH-30/testrepo-Alpha.git";
     protected static final String DISK_REPO_DISPLAY_NAME = "testrepo-Alpha_master";
+    // repos/reposense_testrepo-Alpha/testrepo-Alpha/git-blame-ignore-revs
+    protected static final String IGNORE_REVS_FILE_LOCATION =
+            "repos/FH-30_testrepo-Alpha/testrepo-Alpha/.git-blame-ignore-revs";
```

Comment:
These two variables need to be changed when merging in. It is important that the commit hashes in the branch don't change when somehow adding the branch to testrepo-Alpha (might have to make quite a lot of changes if that's the case)

Reply:
Do you have a PR for this branch?

Reply:
@FH-30 I think you meant to reply to this thread.

> the branch is non existent on the remote repository

I just pushed a new branch `1565-find-previous-authors` (changed the name to be more informative). You can update line 42 to use this name, and submit a PR to that branch.

> I need the commit hashes in my the branch above to be preserved

I will use rebase merging on the command line to preserve the commit hash, so it should not be an issue.

Reply:
I have submitted a PR here:
https://github.com/reposense/testrepo-Alpha/pull/13

Reply:
I have created the commit to change the branch(line 42) and path (line 41) to be consistent with the newly merged PR in testrepo-alpha. However, I will refrain from pushing until it is merged to prevent triggering failing tests on this PR.

Reply:
> I will use rebase merging on the command line to preserve the commit hash, so it should not be an issue.

That's good to hear!

Reply:
@FH-30 For the PR, can you rebase your branch? There were some updates to master, which will cause your commit to not become the latest one in the final branch.

Reply:
@dcshzj I have just rebased it and have done a force push, however it appears that in the process of rebasing the git commit hash gets altered, I'll make the necessary changes in the PR, in the mean time I hope you can merge the branch as soon as possible, thanks!

Reply:
@FH-30 Done! The PR has been merged.

Reply:
@dcshzj the tests are now passing locally, however they aren't passing on the github CI for ubuntu 18.04. I do not have an ubuntu machine so I'm undure about how to fix this, what do you propose?

Reply:
Upon merging with the master branch of RepoSense, it passes. Not entirely sure why.

Reply:
Looks like all good now, probably some hiccup somewhere.

Reply:
Yeah, thanks for the quick review over the past two days. I guess now you can pass it on to the next stage reviewer!